src/test/java/App/TestParser.java

```diff
@@ -0,0 +1,82 @@
+/**
+ * The TestParser class contains JUnit tests for the Parser class.
+ */
+package App;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import java.time.format.DateTimeParseException;
+
+import org.junit.jupiter.api.Test;
+
+import Commands.Command;
+import TaskLists.TaskList;
+import UiRelated.Parser;
+import UiRelated.Ui;
+
+/**
+ * Contains JUnit tests for the Parser class.
+ */
+public class TestParser {
```

Comment:
The guide seems to suggest naming it ParserTest instead? I'm not too sure about this either